sdk/ai/ai-projects/src/evaluations/evaluations.ts

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { Client, createRestError } from "@azure-rest/core-client";
+import { EvaluationOutput, PagedEvaluationOutput } from "../agents/inputOutputs.js";
+import { CreateEvaluationParameters, GetEvaluationParameters, ListEvaluationParameters, UpdateEvaluationParameters } from "./customModels.js";
+
+const expectedGetStatuses = ["200"];
+const expectedCreateStatuses = ["201"];
+
+/** Resource read operation template. */
+export async function getEvaluation(
+    context: Client,
+    evaluationId: string,
+    options?: GetEvaluationParameters,
```

Comment:
These need to be closer to the routes generated for evaluations in the client definitions